deployment/build_and_push.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=$(git rev-parse --short HEAD)
```

Comment:
I use SemVer, git tags and a CHANGELOG.md to track version. If you want a version, use the git tag, if you want a HASH call it a HASH.

Reply:
This is the unique identifier of an image. Can call it hash, np.